src/account/support.ts

```diff
@@ -20,20 +20,6 @@ import { getMainAccount } from "../account";
 import { getAccountBridge } from "../bridge";
 import jsBridges from "../generated/bridge/js";
 
-const experimentalIntegrations = ["tezos"];
```

Comment:
Please just remove tezos, we still need that for incoming Cosmos-JS :pray:

Reply:
Oups there was an env
sorry, was just trigerred by ts i'll find a work around